tests/aws/services/events/test_archive_and_replay.py

```diff
@@ -362,6 +363,10 @@ def test_delete_archive_error_unknown_archive(self, aws_client, snapshot):
 class TestReplay:
     @markers.aws.validated
     @pytest.mark.skipif(is_old_provider(), reason="not supported by the old provider")
+    @pytest.mark.skipif(
+        condition=is_v2_provider(),
```

Comment:
This condition should be on the `EVENT_RULER` config because now the test gets always skipped

Reply:
ideally, this should be:

```
    @pytest.mark.skipif(
        condition=config.EVENT_RULE_ENGINE == "python",
        reason="Not supported with Python-based rule engine",
    )
```

Reply:
(the reason currently contradicts the condition ;)